  INFO root [2019-09-26 10:13:34,970] - 
  INFO root [2019-09-26 10:13:34,970] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:13:35,254] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:13:35,254] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:13:35,255] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:13:35,348] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 10:13:35,349] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 10:13:35,352] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:52762 )
  INFO root [2019-09-26 10:13:35,352] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:52762, SOMAXCONN=128.
  INFO root [2019-09-26 10:13:35,353] - server-configs:
  INFO root [2019-09-26 10:13:35,353] - 	internalTcpAddr : 10.15.76.77:52762
  INFO root [2019-09-26 10:13:35,353] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 10:13:35,353] - 	componentID : 9000
  WARN root [2019-09-26 10:13:35,584] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:13:35,704] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 18840 ----
  INFO root [2019-09-26 10:13:35,705] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:13:35,797] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:13:35,798] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:13:36,205] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:13:36,205] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 10:13:36,382] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:13:36,383] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 10:13:37,404] - ---- loginapp is running ----
 DEBUG root [2019-09-26 10:13:37,404] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:13:44,625] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:13:44,710] - Components::findComponents: found logger, addr:10.15.76.77:52759
  INFO root [2019-09-26 10:13:44,713] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:13:44,714] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:13:44,725] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 10:13:44,815] - Components::findComponents: found dbmgr, addr:10.15.76.77:52760
  INFO root [2019-09-26 10:13:44,816] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:13:44,816] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:13:44,817] - Components::findComponents: found dbmgr, addr:10.15.76.77:52760
  WARN root [2019-09-26 10:13:45,817] - Forwarding logs to logger[10.15.76.77:52759/0/0/0]...
 ERROR root [2019-09-26 10:20:13,022] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3304144980895, lastReceivedTime=3304039988266, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:20:13,036] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:20:13,037] - ==> [2019-09-26 10:20:13 018] TCPPacketReceiver::processPendingEvents(10.15.76.77:52759): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:20:13,037] - ==> [2019-09-26 10:20:13 023] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3304144980895, lastReceivedTime=3304039988266, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-26 10:20:13,120] - TCPPacketReceiver::processPendingEvents(10.15.76.77:52760): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:20:13,133] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3304146089145, lastReceivedTime=3304058379455, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:20:13,171] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 10:20:13,171] - onLoginAppShutDown()
  INFO root [2019-09-26 10:20:13,172] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 10:20:13,172] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 10:20:13,190] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 10:20:13,190] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:20:13,491] - ThreadPool::destroy(): waiting for thread(2)[0x1f54da8,0x1f54ba0,], try=1
  INFO root [2019-09-26 10:37:35,498] - 
  INFO root [2019-09-26 10:37:35,501] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:37:35,502] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:37:35,502] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:37:35,503] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:37:35,546] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 10:37:35,546] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 10:37:35,550] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:52926 )
  INFO root [2019-09-26 10:37:35,550] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:52926, SOMAXCONN=128.
  INFO root [2019-09-26 10:37:35,550] - server-configs:
  INFO root [2019-09-26 10:37:35,550] - 	internalTcpAddr : 10.15.76.77:52926
  INFO root [2019-09-26 10:37:35,550] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 10:37:35,550] - 	componentID : 9000
  WARN root [2019-09-26 10:37:35,767] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:37:36,152] - Components::findLogger: found logger, addr:10.15.76.77:52920
  INFO root [2019-09-26 10:37:36,153] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:37:36,153] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:37:36,159] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 18672 ----
  INFO root [2019-09-26 10:37:36,159] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:37:36,159] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:37:36,160] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:37:36,360] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:37:36,360] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 10:37:36,371] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:37:36,371] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 10:37:36,794] - ---- loginapp is running ----
 DEBUG root [2019-09-26 10:37:36,794] - Components::process(): Request for the process of identity...
  WARN root [2019-09-26 10:37:42,945] - Forwarding logs to logger[10.15.76.77:52920/0/0/0]...
 ERROR root [2019-09-26 10:40:29,454] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3316309295882, lastReceivedTime=3316069025128, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:40:29,471] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:40:29,472] - ==> [2019-09-26 10:40:29 440] TCPPacketReceiver::processPendingEvents(10.15.76.77:52920): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:40:29,472] - ==> [2019-09-26 10:40:29 455] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3316309295882, lastReceivedTime=3316069025128, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-26 10:40:29,721] - TCPPacketReceiver::processPendingEvents(10.15.76.77:52918): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:40:29,728] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3316312034461, lastReceivedTime=3316067146376, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:40:29,731] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 10:40:29,743] - onLoginAppShutDown()
  INFO root [2019-09-26 10:40:29,743] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 10:40:29,744] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 10:40:29,893] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 10:40:29,893] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:40:30,198] - ThreadPool::destroy(): waiting for thread(2)[0x1f984e0,0x1f98548,], try=1
 DEBUG root [2019-09-26 10:40:30,606] - TPThread::~TPThread(): 0x1f984e0
 DEBUG root [2019-09-26 10:40:30,606] - TPThread::~TPThread(): 0x1f98548
 DEBUG root [2019-09-26 10:40:30,606] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-09-26 10:40:30,654] - Bundle::destroyObjPool(): size 20.
 DEBUG root [2019-09-26 10:40:30,654] - Channel::destroyObjPool(): size 14.
 DEBUG root [2019-09-26 10:40:30,655] - TCPPacket::destroyObjPool(): size 3.
 DEBUG root [2019-09-26 10:40:30,655] - UDPPacket::destroyObjPool(): size 2.
 DEBUG root [2019-09-26 10:40:30,655] - EndPoint::destroyObjPool(): size 13.
 DEBUG root [2019-09-26 10:40:30,656] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:40:30,656] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:40:30,656] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-09-26 10:40:30,656] - loginapp(9000) has shut down.
  INFO root [2019-09-26 10:44:55,985] - 
  INFO root [2019-09-26 10:44:55,989] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:44:55,990] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:44:55,990] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:44:55,990] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:44:56,075] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 10:44:56,077] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 10:44:56,084] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53019 )
  INFO root [2019-09-26 10:44:56,085] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53019, SOMAXCONN=128.
  INFO root [2019-09-26 10:44:56,085] - server-configs:
  INFO root [2019-09-26 10:44:56,085] - 	internalTcpAddr : 10.15.76.77:53019
  INFO root [2019-09-26 10:44:56,085] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 10:44:56,085] - 	componentID : 9000
  WARN root [2019-09-26 10:44:56,268] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:44:57,774] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 15400 ----
  INFO root [2019-09-26 10:44:57,778] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:44:57,781] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:44:57,782] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:44:58,030] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:44:58,031] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 10:44:58,043] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:44:58,043] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 10:44:58,558] - ---- loginapp is running ----
 DEBUG root [2019-09-26 10:44:58,558] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:45:05,724] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:45:05,775] - Components::findComponents: found logger, addr:10.15.76.77:53012
  INFO root [2019-09-26 10:45:05,776] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:45:05,776] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:45:05,777] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 10:45:05,789] - Components::findComponents: found logger, addr:10.15.76.77:53012
  INFO root [2019-09-26 10:45:05,793] - Components::findComponents: found dbmgr, addr:10.15.76.77:53016
  INFO root [2019-09-26 10:45:05,793] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:45:05,793] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:45:05,895] - Components::findComponents: found dbmgr, addr:10.15.76.77:53016
  WARN root [2019-09-26 10:45:06,896] - Forwarding logs to logger[10.15.76.77:53012/0/0/0]...
 ERROR root [2019-09-26 10:47:13,904] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3320353795929, lastReceivedTime=3320172205202, inactivityExceptionPeriod=599500640)
  INFO root [2019-09-26 10:47:13,922] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:47:13,922] - ==> [2019-09-26 10:47:13 890] TCPPacketReceiver::processPendingEvents(10.15.76.77:53012): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:47:13,923] - ==> [2019-09-26 10:47:13 905] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3320353795929, lastReceivedTime=3320172205202, inactivityExceptionPeriod=599500640)
  WARN root [2019-09-26 10:47:14,098] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53016): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:47:14,105] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3320355811398, lastReceivedTime=3320175400277, inactivityExceptionPeriod=599500640)
  INFO root [2019-09-26 10:47:14,113] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 10:47:14,116] - onLoginAppShutDown()
  INFO root [2019-09-26 10:47:14,117] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 10:47:14,117] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 10:47:14,131] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 10:47:14,132] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:47:14,433] - ThreadPool::destroy(): waiting for thread(2)[0x19f7d78,0x19f7f18,], try=1
 DEBUG root [2019-09-26 10:47:14,837] - TPThread::~TPThread(): 0x19f7d78
 DEBUG root [2019-09-26 10:47:14,838] - TPThread::~TPThread(): 0x19f7f18
 DEBUG root [2019-09-26 10:47:14,838] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-09-26 10:47:14,875] - Bundle::destroyObjPool(): size 36.
 DEBUG root [2019-09-26 10:47:14,876] - Channel::destroyObjPool(): size 14.
 DEBUG root [2019-09-26 10:47:14,876] - TCPPacket::destroyObjPool(): size 19.
 DEBUG root [2019-09-26 10:47:14,876] - UDPPacket::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:47:14,876] - EndPoint::destroyObjPool(): size 13.
 DEBUG root [2019-09-26 10:47:14,876] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:47:14,876] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-09-26 10:47:14,877] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-09-26 10:47:14,877] - loginapp(9000) has shut down.
  INFO root [2019-09-26 10:53:20,508] - 
  INFO root [2019-09-26 10:53:20,514] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:20,515] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:20,515] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:20,515] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:20,552] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 10:53:20,552] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 10:53:20,555] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53114 )
  INFO root [2019-09-26 10:53:20,556] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53114, SOMAXCONN=128.
  INFO root [2019-09-26 10:53:20,556] - server-configs:
  INFO root [2019-09-26 10:53:20,556] - 	internalTcpAddr : 10.15.76.77:53114
  INFO root [2019-09-26 10:53:20,556] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 10:53:20,556] - 	componentID : 9000
  WARN root [2019-09-26 10:53:20,769] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:22,305] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 19700 ----
  INFO root [2019-09-26 10:53:22,307] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:22,307] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:22,307] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:22,633] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:53:22,633] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 10:53:22,644] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:53:22,644] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 10:53:23,048] - ---- loginapp is running ----
 DEBUG root [2019-09-26 10:53:23,049] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:53:30,205] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:53:30,287] - Components::findComponents: found logger, addr:10.15.76.77:53111
  INFO root [2019-09-26 10:53:30,288] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:53:30,288] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:53:30,290] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-26 10:53:31,393] - Forwarding logs to logger[10.15.76.77:53111/0/0/0]...
 ERROR root [2019-09-26 10:53:37,457] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324189337017, lastReceivedTime=3324117651831, inactivityExceptionPeriod=599498658)
  INFO root [2019-09-26 10:53:37,459] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:53:37,459] - ==> [2019-09-26 10:53:37 458] TCPPacketReceiver::processPendingEvents(10.15.76.77:53111): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:53:37,459] - ==> [2019-09-26 10:53:37 459] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324189337017, lastReceivedTime=3324117651831, inactivityExceptionPeriod=599498658)
  INFO root [2019-09-26 10:53:39,061] - 
  INFO root [2019-09-26 10:53:39,068] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:39,069] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:39,070] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:39,070] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:39,103] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 10:53:39,103] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 10:53:39,112] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53134 )
  INFO root [2019-09-26 10:53:39,112] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53134, SOMAXCONN=128.
  INFO root [2019-09-26 10:53:39,112] - server-configs:
  INFO root [2019-09-26 10:53:39,112] - 	internalTcpAddr : 10.15.76.77:53134
  INFO root [2019-09-26 10:53:39,112] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 10:53:39,112] - 	componentID : 9000
  WARN root [2019-09-26 10:53:39,334] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:40,863] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 18604 ----
  INFO root [2019-09-26 10:53:40,874] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:40,876] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:40,876] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:41,091] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:53:41,092] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 10:53:41,108] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:53:41,109] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 10:53:41,598] - ---- loginapp is running ----
 DEBUG root [2019-09-26 10:53:41,598] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:53:48,769] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:53:48,774] - Components::findComponents: found logger, addr:10.15.76.77:53128
  INFO root [2019-09-26 10:53:48,775] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:53:48,775] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:53:48,776] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-26 10:53:49,879] - Forwarding logs to logger[10.15.76.77:53128/0/0/0]...
 ERROR root [2019-09-26 10:53:54,800] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324362752757, lastReceivedTime=3324302513350, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:53:54,801] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:53:54,801] - ==> [2019-09-26 10:53:54 797] TCPPacketReceiver::processPendingEvents(10.15.76.77:53128): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:53:54,802] - ==> [2019-09-26 10:53:54 800] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324362752757, lastReceivedTime=3324302513350, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:53:54,929] - Components::findComponents: find dbmgr(5)...
  INFO root [2019-09-26 10:53:56,855] - 
  INFO root [2019-09-26 10:53:56,856] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:53:56,858] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:53:56,858] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:53:56,858] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:53:56,901] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 10:53:56,901] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 10:53:56,907] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53145 )
  INFO root [2019-09-26 10:53:56,907] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53145, SOMAXCONN=128.
  INFO root [2019-09-26 10:53:56,907] - server-configs:
  INFO root [2019-09-26 10:53:56,907] - 	internalTcpAddr : 10.15.76.77:53145
  INFO root [2019-09-26 10:53:56,907] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 10:53:56,907] - 	componentID : 9000
  WARN root [2019-09-26 10:53:57,192] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:53:58,733] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 8064 ----
  INFO root [2019-09-26 10:53:58,734] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:53:58,734] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:53:58,735] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:53:58,938] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:53:58,938] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 10:53:58,948] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:53:58,949] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 10:53:59,354] - ---- loginapp is running ----
 DEBUG root [2019-09-26 10:53:59,355] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:54:06,530] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:54:06,593] - Components::findComponents: found logger, addr:10.15.76.77:53146
  INFO root [2019-09-26 10:54:06,594] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:54:06,594] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:54:06,596] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 10:54:06,597] - Components::findComponents: found logger, addr:10.15.76.77:53146
  WARN root [2019-09-26 10:54:07,700] - Forwarding logs to logger[10.15.76.77:53146/0/0/0]...
 ERROR root [2019-09-26 10:54:10,036] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324515126308, lastReceivedTime=3324480716826, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:54:10,039] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:54:10,039] - ==> [2019-09-26 10:54:10 029] TCPPacketReceiver::processPendingEvents(10.15.76.77:53146): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:54:10,040] - ==> [2019-09-26 10:54:10 038] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324515126308, lastReceivedTime=3324480716826, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:54:10,069] - Components::findComponents: find dbmgr(4)...
  INFO root [2019-09-26 10:54:11,931] - 
  INFO root [2019-09-26 10:54:11,932] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:54:11,933] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:54:11,933] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:54:11,933] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:54:11,944] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 10:54:11,944] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 10:54:11,948] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53173 )
  INFO root [2019-09-26 10:54:11,949] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53173, SOMAXCONN=128.
  INFO root [2019-09-26 10:54:11,949] - server-configs:
  INFO root [2019-09-26 10:54:11,949] - 	internalTcpAddr : 10.15.76.77:53173
  INFO root [2019-09-26 10:54:11,949] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 10:54:11,949] - 	componentID : 9000
  WARN root [2019-09-26 10:54:12,122] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 10:54:13,626] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 19404 ----
  INFO root [2019-09-26 10:54:13,635] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:54:13,636] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:54:13,636] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:54:13,900] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:54:13,900] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 10:54:13,912] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:54:13,912] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 10:54:14,339] - ---- loginapp is running ----
 DEBUG root [2019-09-26 10:54:14,339] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:54:21,512] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:54:21,608] - Components::findComponents: found logger, addr:10.15.76.77:53169
  INFO root [2019-09-26 10:54:21,609] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:54:21,609] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:54:21,610] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-26 10:54:22,713] - Forwarding logs to logger[10.15.76.77:53169/0/0/0]...
 ERROR root [2019-09-26 10:54:28,786] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324702613102, lastReceivedTime=3324630853980, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:54:28,787] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:54:28,787] - ==> [2019-09-26 10:54:28 785] TCPPacketReceiver::processPendingEvents(10.15.76.77:53169): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:54:28,787] - ==> [2019-09-26 10:54:28 786] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3324702613102, lastReceivedTime=3324630853980, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 10:56:35,188] - 
  INFO root [2019-09-26 10:56:35,201] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 10:56:35,202] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 10:56:35,203] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 10:56:35,203] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 10:56:35,257] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 10:56:35,257] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 10:56:35,263] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53189 )
  INFO root [2019-09-26 10:56:35,263] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53189, SOMAXCONN=128.
  INFO root [2019-09-26 10:56:35,263] - server-configs:
  INFO root [2019-09-26 10:56:35,263] - 	internalTcpAddr : 10.15.76.77:53189
  INFO root [2019-09-26 10:56:35,264] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 10:56:35,264] - 	componentID : 9000
  WARN root [2019-09-26 10:56:35,490] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-26 10:56:35,856] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20996, errorno=10048
  INFO root [2019-09-26 10:56:35,860] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 16248 ----
  INFO root [2019-09-26 10:56:35,860] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 10:56:35,860] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 10:56:35,861] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 10:56:36,056] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 10:56:36,056] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 10:56:36,069] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 10:56:36,069] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 10:56:36,449] - ---- loginapp is running ----
 DEBUG root [2019-09-26 10:56:36,449] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 10:56:43,629] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 10:56:43,726] - Components::findComponents: found logger, addr:10.15.76.77:53185
  INFO root [2019-09-26 10:56:43,726] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:56:43,727] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:56:43,728] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 10:56:43,830] - Components::findComponents: found dbmgr, addr:10.15.76.77:53182
  INFO root [2019-09-26 10:56:43,830] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 10:56:43,830] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 10:56:43,831] - Components::findComponents: found dbmgr, addr:10.15.76.77:53182
  WARN root [2019-09-26 10:56:44,832] - Forwarding logs to logger[10.15.76.77:53185/0/0/0]...
 ERROR root [2019-09-26 10:58:37,838] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3327193140787, lastReceivedTime=3327166855575, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-26 10:58:37,841] - The following logs sent to logger failed:
  WARN root [2019-09-26 10:58:37,842] - ==> [2019-09-26 10:58:37 831] TCPPacketReceiver::processPendingEvents(10.15.76.77:53185): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:58:37,842] - ==> [2019-09-26 10:58:37 839] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3327193140787, lastReceivedTime=3327166855575, inactivityExceptionPeriod=599499745)
  WARN root [2019-09-26 10:58:37,947] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53182): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 10:58:37,950] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3327194258405, lastReceivedTime=3327165928613, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-26 10:58:37,959] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 10:58:37,960] - onLoginAppShutDown()
  INFO root [2019-09-26 10:58:37,961] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 10:58:37,961] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 10:58:37,978] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 10:58:37,978] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 10:58:38,279] - ThreadPool::destroy(): waiting for thread(2)[0x4a7650,0x4a74b0,], try=1
  INFO root [2019-09-26 11:00:58,103] - 
  INFO root [2019-09-26 11:00:58,104] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:00:58,105] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:00:58,105] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:00:58,105] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:00:58,137] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 11:00:58,138] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 11:00:58,141] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53256 )
  INFO root [2019-09-26 11:00:58,141] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53256, SOMAXCONN=128.
  INFO root [2019-09-26 11:00:58,141] - server-configs:
  INFO root [2019-09-26 11:00:58,141] - 	internalTcpAddr : 10.15.76.77:53256
  INFO root [2019-09-26 11:00:58,141] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 11:00:58,141] - 	componentID : 9000
  WARN root [2019-09-26 11:00:58,335] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:00:59,841] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 6752 ----
  INFO root [2019-09-26 11:00:59,842] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:00:59,842] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:00:59,842] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:01:00,274] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:01:00,275] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 11:01:00,288] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:01:00,289] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 11:01:00,775] - ---- loginapp is running ----
 DEBUG root [2019-09-26 11:01:00,777] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:01:07,949] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:01:07,991] - Components::findComponents: found logger, addr:10.15.76.77:53251
  INFO root [2019-09-26 11:01:07,992] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:01:07,992] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:01:07,993] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:01:07,994] - Components::findComponents: found logger, addr:10.15.76.77:53251
  INFO root [2019-09-26 11:01:07,999] - Components::findComponents: found dbmgr, addr:10.15.76.77:53254
  INFO root [2019-09-26 11:01:07,999] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:01:08,000] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:01:08,101] - Components::findComponents: found dbmgr, addr:10.15.76.77:53254
  WARN root [2019-09-26 11:01:09,101] - Forwarding logs to logger[10.15.76.77:53251/0/0/0]...
 ERROR root [2019-09-26 11:02:47,359] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3329688351850, lastReceivedTime=3329495443475, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-26 11:02:47,364] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:02:47,364] - ==> [2019-09-26 11:02:47 354] TCPPacketReceiver::processPendingEvents(10.15.76.77:53251): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:02:47,364] - ==> [2019-09-26 11:02:47 360] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3329688351850, lastReceivedTime=3329495443475, inactivityExceptionPeriod=599499745)
  WARN root [2019-09-26 11:02:47,464] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53254): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:02:47,476] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3329689523321, lastReceivedTime=3329496195615, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-26 11:02:47,496] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 11:02:47,503] - onLoginAppShutDown()
  INFO root [2019-09-26 11:02:47,503] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:02:47,503] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 11:02:47,512] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 11:02:47,513] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:02:47,814] - ThreadPool::destroy(): waiting for thread(2)[0x1919e50,0x1919eb8,], try=1
  INFO root [2019-09-26 11:02:49,564] - 
  INFO root [2019-09-26 11:02:49,576] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:02:49,577] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:02:49,577] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:02:49,577] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:02:49,641] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 11:02:49,641] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 11:02:49,666] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53311 )
  INFO root [2019-09-26 11:02:49,666] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53311, SOMAXCONN=128.
  INFO root [2019-09-26 11:02:49,667] - server-configs:
  INFO root [2019-09-26 11:02:49,667] - 	internalTcpAddr : 10.15.76.77:53311
  INFO root [2019-09-26 11:02:49,667] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 11:02:49,667] - 	componentID : 9000
  WARN root [2019-09-26 11:02:49,981] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:02:50,195] - Components::findLogger: found logger, addr:10.15.76.77:53306
  INFO root [2019-09-26 11:02:50,195] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:02:50,195] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:02:50,200] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 19620 ----
  INFO root [2019-09-26 11:02:50,201] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:02:50,203] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:02:50,204] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:02:50,419] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:02:50,419] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 11:02:50,433] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:02:50,433] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 11:02:50,893] - ---- loginapp is running ----
 DEBUG root [2019-09-26 11:02:50,894] - Components::process(): Request for the process of identity...
  WARN root [2019-09-26 11:02:57,076] - Forwarding logs to logger[10.15.76.77:53306/0/0/0]...
 ERROR root [2019-09-26 11:04:12,357] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3330538327518, lastReceivedTime=3330309285265, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-26 11:04:12,358] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:04:12,358] - ==> [2019-09-26 11:04:12 356] TCPPacketReceiver::processPendingEvents(10.15.76.77:53306): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:04:12,358] - ==> [2019-09-26 11:04:12 358] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3330538327518, lastReceivedTime=3330309285265, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-26 11:04:12,463] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53309): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:04:12,469] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3330539455775, lastReceivedTime=3330395385776, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-26 11:04:12,485] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 11:04:12,486] - onLoginAppShutDown()
  INFO root [2019-09-26 11:04:12,487] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:04:12,487] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 11:04:12,506] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 11:04:12,507] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:04:12,809] - ThreadPool::destroy(): waiting for thread(2)[0x206bc10,0x206b730,], try=1
  INFO root [2019-09-26 11:06:52,678] - 
  INFO root [2019-09-26 11:06:52,679] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:06:52,680] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:06:52,680] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:06:52,680] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:06:52,696] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 11:06:52,696] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 11:06:52,702] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53384 )
  INFO root [2019-09-26 11:06:52,702] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53384, SOMAXCONN=128.
  INFO root [2019-09-26 11:06:52,702] - server-configs:
  INFO root [2019-09-26 11:06:52,703] - 	internalTcpAddr : 10.15.76.77:53384
  INFO root [2019-09-26 11:06:52,703] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 11:06:52,703] - 	componentID : 9000
  WARN root [2019-09-26 11:06:52,904] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:06:54,444] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 19700 ----
  INFO root [2019-09-26 11:06:54,445] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:06:54,446] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:06:54,446] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:06:54,703] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:06:54,703] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 11:06:54,717] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:06:54,717] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 11:06:55,201] - ---- loginapp is running ----
 DEBUG root [2019-09-26 11:06:55,201] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:07:02,358] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:07:02,458] - Components::findComponents: found logger, addr:10.15.76.77:53383
  INFO root [2019-09-26 11:07:02,458] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:07:02,458] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:07:02,460] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:07:02,562] - Components::findComponents: found dbmgr, addr:10.15.76.77:53381
  INFO root [2019-09-26 11:07:02,562] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:07:02,562] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:07:02,563] - Components::findComponents: found dbmgr, addr:10.15.76.77:53381
  WARN root [2019-09-26 11:07:03,563] - Forwarding logs to logger[10.15.76.77:53383/0/0/0]...
 ERROR root [2019-09-26 11:08:49,376] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3333308516051, lastReceivedTime=3333041708531, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 11:08:49,379] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:08:49,380] - ==> [2019-09-26 11:08:49 373] TCPPacketReceiver::processPendingEvents(10.15.76.77:53383): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:08:49,380] - ==> [2019-09-26 11:08:49 376] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3333308516051, lastReceivedTime=3333041708531, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-26 11:08:49,490] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53381): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:08:49,495] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3333309710434, lastReceivedTime=3333041098710, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-26 11:08:49,510] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 11:08:49,512] - onLoginAppShutDown()
  INFO root [2019-09-26 11:08:49,513] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:08:49,513] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 11:08:49,529] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 11:08:49,529] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:08:49,830] - ThreadPool::destroy(): waiting for thread(2)[0x889f38,0x889fa0,], try=1
  INFO root [2019-09-26 11:08:51,605] - 
  INFO root [2019-09-26 11:08:51,607] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:08:51,608] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:08:51,608] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:08:51,608] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:08:51,659] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 11:08:51,659] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 11:08:51,666] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53459 )
  INFO root [2019-09-26 11:08:51,667] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53459, SOMAXCONN=128.
  INFO root [2019-09-26 11:08:51,667] - server-configs:
  INFO root [2019-09-26 11:08:51,667] - 	internalTcpAddr : 10.15.76.77:53459
  INFO root [2019-09-26 11:08:51,667] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 11:08:51,667] - 	componentID : 9000
  WARN root [2019-09-26 11:08:51,847] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:08:51,956] - Components::findLogger: found logger, addr:10.15.76.77:53452
  INFO root [2019-09-26 11:08:51,956] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:08:51,956] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:08:51,961] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 7112 ----
  INFO root [2019-09-26 11:08:51,961] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:08:51,962] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:08:51,962] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:08:52,161] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:08:52,161] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 11:08:52,174] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:08:52,174] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 11:08:52,611] - ---- loginapp is running ----
 DEBUG root [2019-09-26 11:08:52,612] - Components::process(): Request for the process of identity...
  WARN root [2019-09-26 11:08:58,759] - Forwarding logs to logger[10.15.76.77:53452/0/0/0]...
 ERROR root [2019-09-26 11:10:11,692] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3334131674533, lastReceivedTime=3333927994307, inactivityExceptionPeriod=599498466)
  INFO root [2019-09-26 11:10:11,693] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:10:11,693] - ==> [2019-09-26 11:10:11 690] TCPPacketReceiver::processPendingEvents(10.15.76.77:53452): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:10:11,694] - ==> [2019-09-26 11:10:11 692] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3334131674533, lastReceivedTime=3333927994307, inactivityExceptionPeriod=599498466)
  WARN root [2019-09-26 11:10:11,795] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53454): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:10:11,805] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3334132814020, lastReceivedTime=3333928314558, inactivityExceptionPeriod=599498466)
  INFO root [2019-09-26 11:10:11,812] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 11:10:11,812] - onLoginAppShutDown()
  INFO root [2019-09-26 11:10:11,812] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:10:11,813] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 11:10:11,832] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 11:10:11,833] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:10:12,134] - ThreadPool::destroy(): waiting for thread(2)[0xa287f8,0xa28248,], try=1
  INFO root [2019-09-26 11:10:13,552] - 
  INFO root [2019-09-26 11:10:13,554] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:10:13,555] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:10:13,555] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:10:13,555] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:10:13,564] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 11:10:13,565] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 11:10:13,577] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53491 )
  INFO root [2019-09-26 11:10:13,577] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53491, SOMAXCONN=128.
  INFO root [2019-09-26 11:10:13,577] - server-configs:
  INFO root [2019-09-26 11:10:13,578] - 	internalTcpAddr : 10.15.76.77:53491
  INFO root [2019-09-26 11:10:13,578] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 11:10:13,578] - 	componentID : 9000
  WARN root [2019-09-26 11:10:13,765] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:10:15,280] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 15888 ----
  INFO root [2019-09-26 11:10:15,283] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:10:15,283] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:10:15,284] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:10:15,495] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:10:15,495] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 11:10:15,509] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:10:15,510] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 11:10:15,946] - ---- loginapp is running ----
 DEBUG root [2019-09-26 11:10:15,947] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:10:23,129] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:10:23,216] - Components::findComponents: found logger, addr:10.15.76.77:53490
  INFO root [2019-09-26 11:10:23,217] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:10:23,217] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:10:23,222] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:10:23,326] - Components::findComponents: found dbmgr, addr:10.15.76.77:53493
  INFO root [2019-09-26 11:10:23,326] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:10:23,326] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:10:23,327] - Components::findComponents: found dbmgr, addr:10.15.76.77:53493
  WARN root [2019-09-26 11:10:24,328] - Forwarding logs to logger[10.15.76.77:53490/0/0/0]...
 ERROR root [2019-09-26 11:12:29,803] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3335512794014, lastReceivedTime=3335349695813, inactivityExceptionPeriod=599499041)
  INFO root [2019-09-26 11:12:29,808] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:12:29,809] - ==> [2019-09-26 11:12:29 802] TCPPacketReceiver::processPendingEvents(10.15.76.77:53490): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:12:29,809] - ==> [2019-09-26 11:12:29 804] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3335512794014, lastReceivedTime=3335349695813, inactivityExceptionPeriod=599499041)
  WARN root [2019-09-26 11:12:29,901] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53493): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:12:29,902] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3335513782153, lastReceivedTime=3335373123239, inactivityExceptionPeriod=599499041)
  INFO root [2019-09-26 11:12:29,921] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 11:12:29,925] - onLoginAppShutDown()
  INFO root [2019-09-26 11:12:29,925] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:12:29,926] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 11:12:29,937] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 11:12:29,938] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:12:30,240] - ThreadPool::destroy(): waiting for thread(2)[0x1e94bb0,0x1e94c18,], try=1
  INFO root [2019-09-26 11:12:31,557] - 
  INFO root [2019-09-26 11:12:31,560] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:12:31,561] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:12:31,561] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:12:31,561] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:12:31,603] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 11:12:31,603] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 11:12:31,608] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53573 )
  INFO root [2019-09-26 11:12:31,608] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53573, SOMAXCONN=128.
  INFO root [2019-09-26 11:12:31,608] - server-configs:
  INFO root [2019-09-26 11:12:31,608] - 	internalTcpAddr : 10.15.76.77:53573
  INFO root [2019-09-26 11:12:31,608] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 11:12:31,608] - 	componentID : 9000
  WARN root [2019-09-26 11:12:31,815] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:12:33,319] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 16632 ----
  INFO root [2019-09-26 11:12:33,320] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:12:33,320] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:12:33,321] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:12:33,584] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:12:33,584] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 11:12:33,595] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:12:33,596] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 11:12:34,031] - ---- loginapp is running ----
 DEBUG root [2019-09-26 11:12:34,031] - Components::process(): Request for the process of identity...
  INFO root [2019-09-26 11:12:41,179] - Components::findComponents: find logger(1)...
  INFO root [2019-09-26 11:12:41,212] - Components::findComponents: found logger, addr:10.15.76.77:53572
  INFO root [2019-09-26 11:12:41,212] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:12:41,212] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:12:41,213] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-26 11:12:41,322] - Components::findComponents: found dbmgr, addr:10.15.76.77:53566
  INFO root [2019-09-26 11:12:41,323] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:12:41,323] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:12:41,325] - Components::findComponents: found dbmgr, addr:10.15.76.77:53566
  WARN root [2019-09-26 11:12:42,325] - Forwarding logs to logger[10.15.76.77:53572/0/0/0]...
 ERROR root [2019-09-26 11:15:24,609] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3337260844660, lastReceivedTime=3337030909057, inactivityExceptionPeriod=599498594)
  INFO root [2019-09-26 11:15:24,612] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:15:24,612] - ==> [2019-09-26 11:15:24 607] TCPPacketReceiver::processPendingEvents(10.15.76.77:53572): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:15:24,612] - ==> [2019-09-26 11:15:24 609] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3337260844660, lastReceivedTime=3337030909057, inactivityExceptionPeriod=599498594)
  WARN root [2019-09-26 11:15:24,699] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53566): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:15:24,700] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3337261765196, lastReceivedTime=3337030402498, inactivityExceptionPeriod=599498594)
  INFO root [2019-09-26 11:15:24,706] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 11:15:24,719] - onLoginAppShutDown()
  INFO root [2019-09-26 11:15:24,719] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:15:24,719] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 11:15:24,733] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 11:15:24,733] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:15:25,034] - ThreadPool::destroy(): waiting for thread(2)[0x1e08a50,0x1e08b20,], try=1
  INFO root [2019-09-26 11:15:26,496] - 
  INFO root [2019-09-26 11:15:26,499] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:15:26,500] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:15:26,501] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:15:26,501] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:15:26,515] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 11:15:26,515] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 11:15:26,520] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53634 )
  INFO root [2019-09-26 11:15:26,521] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53634, SOMAXCONN=128.
  INFO root [2019-09-26 11:15:26,521] - server-configs:
  INFO root [2019-09-26 11:15:26,521] - 	internalTcpAddr : 10.15.76.77:53634
  INFO root [2019-09-26 11:15:26,521] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 11:15:26,521] - 	componentID : 9000
  WARN root [2019-09-26 11:15:26,699] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:15:27,031] - Components::findLogger: found logger, addr:10.15.76.77:53628
  INFO root [2019-09-26 11:15:27,031] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:15:27,031] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:15:27,036] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 7268 ----
  INFO root [2019-09-26 11:15:27,036] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:15:27,037] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:15:27,037] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:15:27,237] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:15:27,237] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 11:15:27,249] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:15:27,250] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 11:15:27,649] - ---- loginapp is running ----
 DEBUG root [2019-09-26 11:15:27,649] - Components::process(): Request for the process of identity...
  WARN root [2019-09-26 11:15:33,779] - Forwarding logs to logger[10.15.76.77:53628/0/0/0]...
 ERROR root [2019-09-26 11:18:22,248] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3339037242124, lastReceivedTime=3338778396969, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 11:18:22,253] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:18:22,253] - ==> [2019-09-26 11:18:22 246] TCPPacketReceiver::processPendingEvents(10.15.76.77:53628): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:18:22,254] - ==> [2019-09-26 11:18:22 249] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3339037242124, lastReceivedTime=3338778396969, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-26 11:18:22,360] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53629): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:18:22,362] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3339038378318, lastReceivedTime=3338778961940, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 11:18:22,376] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 11:18:22,386] - onLoginAppShutDown()
  INFO root [2019-09-26 11:18:22,386] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:18:22,386] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 11:18:22,401] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 11:18:22,402] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:18:22,703] - ThreadPool::destroy(): waiting for thread(2)[0x847270,0x8473a8,], try=1
  INFO root [2019-09-26 11:19:33,060] - 
  INFO root [2019-09-26 11:19:33,064] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-26 11:19:33,065] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-26 11:19:33,065] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-26 11:19:33,065] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-26 11:19:33,089] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:20013 )
  INFO root [2019-09-26 11:19:33,090] - NetworkInterface::initialize(EXTERNAL-TCP): address 10.15.76.77:20013, SOMAXCONN=128.
  INFO root [2019-09-26 11:19:33,093] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 10.15.76.77 ( 10.15.76.77:53703 )
  INFO root [2019-09-26 11:19:33,094] - NetworkInterface::initialize(INTERNAL-TCP): address 10.15.76.77:53703, SOMAXCONN=128.
  INFO root [2019-09-26 11:19:33,094] - server-configs:
  INFO root [2019-09-26 11:19:33,094] - 	internalTcpAddr : 10.15.76.77:53703
  INFO root [2019-09-26 11:19:33,094] - 	externalTcpAddr : 10.15.76.77:20013
  INFO root [2019-09-26 11:19:33,094] - 	componentID : 9000
  WARN root [2019-09-26 11:19:33,298] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-26 11:19:33,430] - Components::findLogger: found logger, addr:10.15.76.77:53696
  INFO root [2019-09-26 11:19:33,430] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-26 11:19:33,431] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-26 11:19:33,440] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 41181. PID: 6008 ----
  INFO root [2019-09-26 11:19:33,442] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-26 11:19:33,443] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-26 11:19:33,444] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-26 11:19:33,655] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-26 11:19:33,655] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-26 11:19:33,671] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-26 11:19:33,671] - TelnetServer server is running on port 31000
  INFO root [2019-09-26 11:19:34,117] - ---- loginapp is running ----
 DEBUG root [2019-09-26 11:19:34,118] - Components::process(): Request for the process of identity...
  WARN root [2019-09-26 11:19:40,274] - Forwarding logs to logger[10.15.76.77:53696/0/0/0]...
 ERROR root [2019-09-26 11:21:25,811] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3340872870169, lastReceivedTime=3340642847439, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 11:21:25,814] - The following logs sent to logger failed:
  WARN root [2019-09-26 11:21:25,814] - ==> [2019-09-26 11:21:25 807] TCPPacketReceiver::processPendingEvents(10.15.76.77:53696): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:21:25,815] - ==> [2019-09-26 11:21:25 812] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3340872870169, lastReceivedTime=3340642847439, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-26 11:21:25,913] - TCPPacketReceiver::processPendingEvents(10.15.76.77:53701): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-26 11:21:25,916] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=3340873918658, lastReceivedTime=3340644394115, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-26 11:21:25,951] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2019-09-26 11:21:25,961] - onLoginAppShutDown()
  INFO root [2019-09-26 11:21:25,961] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-26 11:21:25,961] - Shutdowner::onShutdownEnd: shutting down
  INFO root [2019-09-26 11:21:25,980] - Script::uninstall(): is successfully!
 DEBUG root [2019-09-26 11:21:25,980] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-26 11:21:26,281] - ThreadPool::destroy(): waiting for thread(2)[0x1d38c58,0x1d38710,], try=1
